src/main/java/thePackmaster/patches/CompendiumPatches.java

```diff
@@ -131,4 +138,28 @@ public static void packSort(CardLibSortHeader input, boolean isAscending) {
             );
         }
     }
+
+    @SpirePatch2(clz = EverythingFix.Initialize.class, method = "Insert")
+    public static class ShowBaseGameCards {
+        @SpirePostfixPatch
+        public static void showBaseGameCards(Object __obj_instance) {
+            if (EverythingFix.Fields.cardGroupMap.containsKey(ThePackmaster.Enums.PACKMASTER_RAINBOW)) {
+                CardGroup group = EverythingFix.Fields.cardGroupMap.get(ThePackmaster.Enums.PACKMASTER_RAINBOW);
+                List<AbstractCard> baseGameCards = CardLibrary.getAllCards().stream()
+                        .filter(c -> (c.color == AbstractCard.CardColor.RED || c.color == AbstractCard.CardColor.GREEN || c.color == AbstractCard.CardColor.BLUE || c.color == AbstractCard.CardColor.PURPLE))
```

Comment:
In theory we could allow people to add colorless cards to packs as well, but to my knowledge nobody has done that and I thought it was better to start with a more restrictive check here.